docs/Inventory-of-Epics.md

```diff
@@ -0,0 +1,27 @@
+# "Inventory of Epics" for the Roslyn Analyzer Ecosystem
+
+Here is a current list of the major areas of work ("epics") for the Roslyn analyzer ecosystem. See the [README](https://github.com/dotnet/roslyn-analyzers-contrib/blob/master/README.md) for an explanation of what that means.
+
+1. Replace FxCop
```

Comment:
It's probably worth noting that this is for VS. Binray analysis is likely useful outside of VS where sources aren't available.